openpype/modules/ftrack/plugins/publish/integrate_ftrack_instances.py

```diff
@@ -10,6 +10,7 @@
 )
 from openpype.lib.profiles_filtering import filter_profiles
 from openpype.lib.transcoding import VIDEO_EXTENSIONS
+from openpype.plugins.publish.integrate import get_representation_path
```

Comment:
Even though it's good idea there are few issues with this approach. We don't import from other plugins, because they are dynamically imported, so if there is common logic for multiple plugins it should not be inside plugin file but elsewhere. Then the function name `get_representation_path` is already reserved for loading. Another issue is that the function does only part of the job from what the name says, it actually returns only first file path and nothing else which may be confusing to be an "api" function.

If the function should stay in core (and I don't know if should) it should be renamed to something like `get_publish_repre_path` and moved to e.g. `openpype/pipeline/publish/utils.py`.

Reply:
Yeah fair enough, I placed it there with hesitation knowing I wanted to discuss it with you on where it should really go, doing it now!